modules/nf-core/preseq/lcextrap/main.nf

```diff
@@ -1,9 +1,9 @@
 process PRESEQ_LCEXTRAP {
     tag "$meta.id"
     label 'process_single'
-    label 'error_ignore'
+    label 'error_retry'
```

Comment:
I think error_ignore was a good idea actually

Reply:
Check line 24 - if retrying it uses a different flag.